src/main/java/beans/ParticipantBean.java

```diff
@@ -1,12 +1,17 @@
 package beans;
 
-public class ParticipantBean {
+import java.io.Serializable;
+
+import helper.BeanHelper;
+
+public class ParticipantBean extends BeanHelper implements Serializable {
 	
 	public static final long serialVersionUID = 1L;
 	
 	private int eventId;
 	private int userId;
 	private int statusId;
+	private int action;
```

Comment:
If this is the same value as the ones stated in the enum you can replace the int with the Enum itself, if I am not mistaken gson converts the type automatically as long as we have the same enum used on our fronted side.